### DISCUSSION TITLE
Regarding Fork the Repo

### DISCUSSION THREAD

**Discussion by nitinNT** [avinashkranjan/Recess#73] [Q&A] [answered]
can you once check the contributing documentation? In that section, if you already Forked the project, which checkout branch should we mention in that and others can you please tell me a format for naming the branch name for our PR.

Please Update.

**Comment by avinashkranjan** [comment, marked as answer]
Hey @nitinNT, 
- You are free to choose any name for your branch there's no restrictions there. 
- That is only to update your local copy of the project with the original copy, with the branch name say master or the branch you are working on has some changes which is not pulled/ If this is too confusing for you just delete and fork the project again when you want to create a PR. Else you can check GitHub official documentation [here](https://towardsdatascience.com/getting-started-with-git-and-github-6fcd0f2d4ac6)
